File: Java/1164.java
import java.util.Scanner;

public class Main{
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int car[] = new int[3];
        
        for(int i = 0; i < 3; i++){
            car[i] = sc.nextInt();
        }
        
        int s = 1;
        
        for(int i = 0; i < 3; i++){;
        	if(car[i]<=170){
        		s = 0;
        		break;
    		}
    	}
    	
    	if(s == 1){
    	    System.out.printf("PASS");
    	} else{
    	    System.out.printf("CRASH");
    	}
    }
}

File: C/1164.c
#include <stdio.h>

int main(){
    int car[3], s;
    scanf("%d %d %d", &car[0], &car[1], &car[2]);
    
    for(int i = 0; i < 3; i++){
    	s = 1;
    	if(car[i]<=170){
    		s = 0;
    		break;
		}
	}
	
	if(s) printf("PASS");
	else printf("CRASH");
	return 0;
}